R/rapwhale-package.R

```diff
@@ -0,0 +1,127 @@
+#' @keywords internal
+"_PACKAGE"
+
+## usethis namespace: start
+#' @importFrom colorspace coords
+#' @importFrom dplyr across
+#' @importFrom dplyr add_count
+#' @importFrom dplyr anti_join
+#' @importFrom dplyr arrange
+#' @importFrom dplyr bind_cols
+#' @importFrom dplyr bind_rows
+#' @importFrom dplyr case_when
+#' @importFrom dplyr count
+#' @importFrom dplyr desc
+#' @importFrom dplyr distinct
+#' @importFrom dplyr filter
+#' @importFrom dplyr full_join
+#' @importFrom dplyr group_by
+#' @importFrom dplyr group_cols
+#' @importFrom dplyr group_indices
+#' @importFrom dplyr group_size
+#' @importFrom dplyr group_vars
+#' @importFrom dplyr groups
+#' @importFrom dplyr if_else
+#' @importFrom dplyr intersect
```

Comment:
Har me eksempel på at me brukar `intersect()` frå dplyr (i staden for frå base)?

Reply:
Nei, det har me kanskje ikkje, så den kan kanskje fjernast. Eg tenkte berre at sidan me tidlegare hadde importert heile `dplyr`-pakken, så la eg til denne linja for at det skulle vera identisk som tidlegare.

Reply:
Fjerna.